.schema/api.openapi.json

```diff
@@ -1932,6 +1932,81 @@
         ]
       }
     },
+    "/self-service/saml/metadata":{
+      "get":{
+        "description": "This endpoint is for the IDP to obtain kratos metadata",
+        "operationId": "getSamlMetadata",
+        "response": {
```

Comment:
I'm missing a few bits on SAML flow, so this might be an irrelevant comment; is it expected there's no 200 response ?